capacity_range error description for CreateVolumeRequest was impoved (#436)
container-storage-interface/spec: spec.md

| Source does not exist | 5 NOT_FOUND | Indicates that the specified source does not exist. | Caller MUST verify that the `volume_content_source` is correct, the source is accessible, and has not been deleted before retrying with exponential back off. |
| Volume already exists but is incompatible | 6 ALREADY_EXISTS | Indicates that a volume corresponding to the specified volume `name` already exists but is incompatible with the specified `capacity_range`, `volume_capabilities` or `parameters`. | Caller MUST fix the arguments or use a different `name` before retrying. |
| Unable to provision in `accessible_topology` | 8 RESOURCE_EXHAUSTED | Indicates that although the `accessible_topology` field is valid, a new volume can not be provisioned with the specified topology constraints. More human-readable information MAY be provided in the gRPC `status.message` field. | Caller MUST ensure that whatever is preventing volumes from being provisioned in the specified location (e.g. quota issues) is addressed before retrying with exponential backoff. |

> | Unsupported `capacity_range` | 11 OUT_OF_RANGE | Indicates that the capacity range is not allowed by the Plugin, for example when trying to create a volume smaller than the source snapshot or when a plugin does not have defaults to handle the request with `capacity_range` not having being set. More human-readable information MAY be provided in the gRPC `status.message` field. | Caller MUST fix the capacity range before retrying or MUST ensure plugin for being reconfigured with meaningful defaults. |

Reviewer: ```suggestion
| Unsupported `capacity_range` | 11 OUT_OF_RANGE | Indicates the specified `capacity_range` is not allowed by the SP (for example when trying to create a volume smaller than the source snapshot) or `capacity_range` is not specified while the SP requires it. More human-readable information MAY be provided in the gRPC `status.message` field. | Caller MUST fix the capacity range before retrying. |
```
